### DISCUSSION TITLE
How to claim?

### DISCUSSION THREAD

**Discussion by bmccafferty** [ansible-community/ansible-badges#4] [Q&A]
Hey,

I just did my very first Issue and PR that was accepted for the ansible servicenow.itsm collection.  Was not sure if that was a valid use case for a badge to be issued.

Just wondered what the process would be to claim if it was?

Issue:
https://github.com/ansible-collections/servicenow.itsm/issues/168

PR:
https://github.com/ansible-collections/servicenow.itsm/pull/169

Thanks in advance.

Brian

**Comment by gundalow** [comment]
@bmccafferty I collections being included into the `ansible` package is a huge milestone, and should be celebrated. 

@Ompragash Could we get badges designed for this? I can work with you to identify the people to award the badges to.

**Reply by Ompragash** [reply]
@bmccafferty Congrats on your very first contribution to Ansible `servicenow.itsm` collection!🎉

@gundalow I designed badges for GitHub contributions but we can surely design one for collection inclusion to Ansible package.

**Reply by bmccafferty** [reply]
Thanks everyone :)

Just love me a digital badge :)
